src/main/java/me/rigamortis/seppuku/impl/module/movement/PullDownModule.java

```diff
@@ -0,0 +1,51 @@
+package me.rigamortis.seppuku.impl.module.movement;
+
+import me.rigamortis.seppuku.api.event.EventStageable;
+import me.rigamortis.seppuku.api.event.player.EventPlayerUpdate;
+import me.rigamortis.seppuku.api.module.Module;
+import me.rigamortis.seppuku.api.value.NumberValue;
+import net.minecraft.client.Minecraft;
+import team.stiff.pomelo.impl.annotated.handler.annotation.Listener;
+
+/**
+ * @author cookiedragon234
+ * (cookiedragon234 owns me and all)
+ * also buy backdoored client kk thx
+ */
+public class PullDownModule extends Module
+{
+	public PullDownModule()
+	{
+		super("PullDown", new String[]{"FastFall"}, "Increase your downwards velocity when falling", "NONE", -1, ModuleType.MOVEMENT);
+	}
+	
+	public final NumberValue<Float> speed = new NumberValue<>("Speed", new String[]{"velocity"}, 10f, Float.class, 0f, 20f, 1f);
```

Comment:
Please follow the format of other module implementations and place members above the initial constructor of the class.

Reply:
Done